### ISSUE TITLE
Maybe UCID a better name?

### ISSUE THREAD

**Issue by rafaelsales** [alizain/ulid#4]
This guy has a good point:
> Sorry, erm.... What makes you say UUIDs are not lexicographically sortable?
They're not chronologically sortable, sure - but I'm confused by your choice of names.

**U**nique **C**hronologically sortable **Id**entifier

**Comment by alizain** [comment]
Yes I had thought of UCID as well, but I didn't like the sound of it, "you-lid" just flowed off the tongue nicely. And the intent was that by sorting ULIDs lexicographically, you're getting valuable information, unlike UUIDs
